widget/view.controller.js

```diff
@@ -214,10 +298,10 @@
 
     function indicatorTypeChanged(iocType) {
       $scope.iocTypeSelected = true;
-      if (iocType === 'Add Custom Indicator Type') {
+      if (iocType === $scope.viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_ADD_IOC_TYPE_CUSTOM_IOC_LABEL) {
```

Comment:
IMO - for comparison we won't require translated label

Reply:
Made necessary changes